### ISSUE TITLE
en-GB locale not accepting query parameters for donation form

### ISSUE THREAD

**Issue by mtdenton** [MozillaFoundation/donate-wagtail#1695]
**Describe the bug**
The en-GB locale specifically isn't allowing values on the donate form to be updated through query parameters. Example link:
https://donate.mozilla.org/en-GB/?presets=33,27,24,21&amount=33&currency=usd

For other locales, this will change the presets and set the amount as default. This may not be the only locale affected but it's the only one I've found so far.

**Expected behavior**
Donation amounts, currency type, selected amount should change based on their respective values in the query string, see this page for correct functionality:

https://donate.mozilla.org/en-US/?presets=33,27,24,21&amount=33&currency=gbp

**Comment by danielfmiranda** [comment]
Added the blocked ticket as this tickets needs some more discussion:

I have given the code a look through, and tried to troubleshoot the error on staging and my local machine, but have not been able to reproduce the bug.

I also connected to the Prod DB and tested on my machine to see if this error was possibly caused by a setting or the data on production, but was still unable to reproduce.

When bringing this up during todays stand up, Jen  mentioned that there may be a WAF rule in place that is causing this, as the /en-GB/ route in particular was being attacked a couple of months ago.
